packages/blob-propagator/test/gcs-worker.test.ts

```diff
@@ -0,0 +1,28 @@
+import { afterAll, beforeAll, describe, vi } from "vitest";
+
+import { runStorageWorkerTestSuite } from "./helpers";
+
+describe(
+  "GCS Worker",
+  runStorageWorkerTestSuite("GOOGLE", {
+    runBeforeAllFns() {
+      beforeAll(() => {
+        vi.stubEnv("CHAIN_ID", "70118930558");
+        vi.stubEnv("GOOGLE_STORAGE_ENABLED", "false");
```

Comment:
```suggestion
        vi.stubEnv("GOOGLE_STORAGE_ENABLED", "true");
```

Reply:
Having a closer look at the tests. I believe we are encountering the following issue: https://github.com/vitest-dev/vitest/issues/2752